resources/js/app.js

```diff
@@ -4,18 +4,19 @@
 	app.config(['$routeProvider',
 		function($routeProvider) {
```

Comment:
hey could you remove the ``:`` from commit message? it only counts if you remove it for some reason :)